incubator/appdaemon/0.1.0/test_values.yaml

```
@@ -0,0 +1,52 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  HA_URL: "http://192.168.5.31:8123"
+  # used to point appdaemon to your home assistant instance
+  TOKEN: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJjNjNiMzZkM2QwMTg0M2Y4OGQ2Y2JkYjY4MmViNGQ2NiIsImlhdCI6MTYxODk0MDc4OSwiZXhwIjoxOTM0MzAwNzg5fQ.CUXm9K4jSS--qBQZ9DahqFL1YIgIcCK32pfORqiQ0ns"
+  # used to authenticate app daemon to your home assistant instance
+  DASH_URL: "localhost:5050"
+  # url for the appdaemon dashboard, an alternative interface for home assistant
+  TZ: "America/Chicago"
```

@Ornias1993 I thought I had a decent understanding of YAML but nonetheless, I will do some additional reading.

In commit  f570344 I removed the indent but the test is back to complaining about the timezone not being set. Obviously, I'm missing something, I'll keep learning. Thanks for all your help with this pr so far!

```suggestion
TZ: "America/Chicago"
```

---

It might be me missing something actually, holdon! :)

---

Got it:
The correct setting would be:

```timezone: "America/Chicago"```
This should translate to TZ inside the container.